lib/msal-common/src/authority/Authority.ts

```diff
@@ -552,13 +552,15 @@ export class Authority {
                 : [];
             if (metadata.length === 0) {
                 // If no metadata is returned, authority is untrusted
+                this.logger.verbose("The network request completed successfully, but no metadata was returned.");
                 return null;
             }
             match = Authority.getCloudDiscoveryMetadataFromNetworkResponse(
                 metadata,
                 this.hostnameAndPort
             );
         } catch (e) {
+            this.logger.verbose("There was a network error while attempting to get the cloud discovery metadata.");
```

Comment:
Can you please log this error, otherwise there is no way to understand what's happening.

Reply:
Alos, use Error verbosity i.e. `this.logger.error(...)`

Reply:
I was thinking about using .error, but we already use a .error when we throw the untrusted authority error. Do you think we should use it here too?

Reply:
This block swallows the original exception (e), making the root cause impossible to determine. The root cause should be logged at error level.